hack/manifests/csi/vsphere/02-example-namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: windows-webserver
```

Comment:
Not sure about `windows-webserver`  as the correct name for the namespace. Looking at the pod, it does not run a webserver. Consider something like: `example-windows`

Reply:
`windows-storage-example` ?